### PR TITLE
feat: Update plugin registration workflow

### PR DESCRIPTION
**Pull request by SilviaZeta**
## Description
Update the workflow for registering a new plugin, as per gliff-ai/manage#220.

Note:
- edit and delete functionalities will go in the next PR.
- the "Enabled" column and any other part not included in the design are (more) temporary.

relates gliff-ai/manage#220

## Checklist:

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request). If you're unsure about any of these, don't hesitate to leave a comment on this pull request!

- [ ] I have read the gliff.ai Contribution Guide.
- [ ] I have requested to **pull a branch** and not from main.
- [ ] I have checked all commit message styles match the requested structure.
- [ ] My code follows the style guidelines of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have performed a self-review of my own code.
- [ ] I have assigned **3 or less** reviewers.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.
- [ ] If appropriate, I have bumped any version numbers.

### CODE REVIEW

**Review by joshuajames-smith — CHANGES_REQUESTED** [review]
Happy to leave the new columns as is for now – give me some time to have a think and design! Same with selecting which platform the plugin is in – this needs some thinking and design, so will work for now!

Just a couple small changes I'd like to make to polish the work and progress you've done!

- [x] 1- Could the spacing on `first line of text` have the layout balanced better visually? So the gap top and bottom is even?

**current design:**
Screenshot 2022-01-26 at 11 14 32

**intended design:**
Screenshot 2022-01-26 at 11 17 48

- [x] 2- Can these `text fields` be within a rounded box?

**current design:**
Screenshot 2022-01-26 at 11 25 44

**intended design:**
Screenshot 2022-01-26 at 11 28 09

- [x] 3- There is some visual inconsistency between `buttons` – flat vs. drop-shadow. I think my future aim is the have these flat, but happy for now to use drop-shadow for platform consistency and leave this for another issue?

**current design:**
Screenshot 2022-01-26 at 11 29 50
Screenshot 2022-01-26 at 11 34 21

- [x] 4- The `heading text` and `dialogue content` should all align evenly on the left.

**current design:**
Screenshot 2022-01-26 at 11 44 49

**intended design:**
Screenshot 2022-01-26 at 11 46 58

- [ ] 5- When a `check-box` is ticked, the green box is smaller? Not sure if this is an icon.svg issue or is fixable? It should be the same size as the black-outline box ideally.

**current design:**
Screenshot 2022-01-26 at 11 49 00

- [x] 6?- In the `drop-down menu`, each row's hover state is green with the selected rows being grey. The original intention was the grey was hover state – I made a similar comment in #245. This would work for now, and maybe require me to think a bit more about the `drop-down menu` design if we think we need this extra visual level?

**current design:**
Screenshot 2022-01-26 at 11 42 08

**Comment by SilviaZeta** [comment]
fixed all styling issues except for n.5, for which I think I'd need a slightly different icon with no box outline?

**Review by cooper667 — APPROVED** [review]

**Comment by joshuajames-smith** [comment]
@SilviaZeta has the icon helped to sort n.5?

File: .github/workflows/pr-title-labeler.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 on:
+  push:
+    branches:
+      - main
   pull_request:
-    types: [opened, edited, synchronize]
+    types: [opened, edited, synchronize, auto_merge_enabled, reopened]
     branches:
-      - main, staging
+      - main
```
Comment (ChasNelson1990) [inline .github/workflows/pr-title-labeler.yml:9]: this is not quite right (see other PR)

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-title-labeler.yml (hunk 1 above)
Comment (SilviaZeta) [reply .github/workflows/pr-title-labeler.yml:9]: Is that necessary given that MANAGE doesn't have a staging branch?

Comment (ChasNelson1990) [reply .github/workflows/pr-title-labeler.yml:9]: .... touche!

**Review by ChasNelson1990 — DISMISSED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Review by philipjackson — APPROVED** [review]

**Review by joshuajames-smith — APPROVED** [review]

**Pull request merged**